### ISSUE TITLE
[Bug]: Combobox does not have `aria-invalid` attr

### ISSUE THREAD

**Issue by Thomas-Neuhauser** [microsoft/fluentui#33490]
### Package

react

### Package version

8.120.5

### React version

16.14.0

### Environment

```shell
System:
    OS: Windows 11 10.0.22631
    CPU: (16) x64 Intel(R) Core(TM) i9-10885H CPU @ 2.40GHz
    Memory: 16.51 GB / 31.72 GB
  Browsers:
    Edge: Chromium (127.0.2651.86)
    Internet Explorer: 11.0.22621.3527
```

### Current Behavior

We are facing accessibility issues using the ComboBox control.
This control is not setting an aria-invalid attribute corresponding to the presence of an error message set via errorMessage.
You can check the render output and behavior here:
https://developer.microsoft.com/en-us/fluentui#/controls/web/combobox

If you check the same for a DropDown or TextField you can see, that these controls always set an aria-invalid attribute to either false or true:
https://developer.microsoft.com/en-us/fluentui#/controls/web/dropdown
https://developer.microsoft.com/en-us/fluentui#/controls/web/textfield

### Expected Behavior

All controls should consistently use the combination of aria-describedby, role="alert" and aria-invalid in order to let screen readers correctly react to error situations.
For the ComboBox we built a workaround using autofill for setting aria-invalid at the inner input field depending on the presence of an error message for the time being.

### Reproduction

https://developer.microsoft.com/en-us/fluentui#/controls/web/combobox

### Steps to reproduce

1. Open https://developer.microsoft.com/en-us/fluentui#/controls/web/combobox
2. In the page search for "ComboBox with static error message"
3. Open the debugger and check the render output for the input field part of the ComboBox
4. Notice that there is no aria-invalid="false" attribute
5. Open https://developer.microsoft.com/en-us/fluentui#/controls/web/dropdown
6. In the page search for "Dropdown with error message"
7. Open the debugger and check the render output for the span part of the DropDown
8. Switch the error on and off, notice that there is an aria-invalid attribute switched between true / false correspondingly

### Are you reporting an Accessibility issue?

yes

### Suggested severity

Medium - Has workaround

### Are you willing to submit a PR to fix?

no

### Validations

- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [x] The provided reproduction is a minimal reproducible example of the bug.

**Comment by layershifter** [comment]
<img width="748" alt="Image" src="https://github.com/user-attachments/assets/6a7530b8-88d3-49d4-a347-d77fb4863fb7" />

I think that it's intentional as there is a separate error message element linked via `aria-describedby`, but I will let @smhigley to check 🐱

**Comment by Thomas-Neuhauser** [comment]
I don't think this makes sense. If there is an error, there should be an aria-invalid indicating this.
At least we get the feedback from our ACC testers that the current behavior doesn't work for them.
If they enter something invalid in a ComboBox and tab out of the field, the screenreader does not react to the error situation and read out the error.

**Comment by layershifter** [comment]
@Thomas-Neuhauser yes, sorry I noticed it already. I updated my message 👍

**Comment by Thomas-Neuhauser** [comment]
Ok I'm sorry now if my response doesn't fit exactly to what you wrote, since you finally now state "Looks like a bug..."
I responded to your original comment which I got notified about via mail but I see you edited it, so sorry if this is confusing now ;-)

**Comment by smhigley** [comment]
Oh yeah, that's definitely a bug, thanks for finding it! Looks like the v8 Dropdown is actually also not setting `aria-invalid` correctly -- it should go on the element with `role=combobox`, it does nothing on the `<span>` that currently has it 😅.

I'll make a PR that fixes both. Thanks again!

**Comment by Thomas-Neuhauser** [comment]
Hey @smhigley thanks for fixing it :-)
Could you please let me know, with which fluentui version the fix will be available?
The milestone of the PR just says January project cycle Q1 2025.
Thanks!

**Comment by khmakoto** [comment]
Hey @Thomas-Neuhauser, this should be available as part of `@fluentui/react-components` version `9.57.0`.

**Comment by Thomas-Neuhauser** [comment]
Hello @khmakoto so there will be no correction in FluentUI version 8?

**Comment by Thomas-Neuhauser** [comment]
Hello, just asking again shorty, if this fix will also be available in FluentUI version 8.
We have our workaround running but I'd like to add in our backlog if this is going to be available in FluentUI 8 with some release, so that we have this for later reference.

**Comment by khmakoto** [comment]
Hey @Thomas-Neuhauser, sorry, I got the wrong PR number at the beginning, this should be available starting in version `8.122.4` of `@fluentui/react`.

**Comment by Thomas-Neuhauser** [comment]
Hi @khmakoto perfect, thanks for the fix and the information :-)
